pkg/daemon/ceph/osd/volume.go

```diff
@@ -79,9 +80,20 @@ func (a *OsdAgent) initializeDevices(context *clusterd.Context, devices *DeviceO
 		strconv.Itoa(a.storeConfig.OSDsPerDevice),
 	}...)
 
-	// ceph-volume is soon implementing a parameter to specify the "fast devices", which correspond to the "metadataDevice" from the
-	// crd spec. After that is implemented, we can implement this. In the meantime, we fall back to use rook's partitioning.
+	if a.storeConfig.StoreType == config.Bluestore && a.storeConfig.DatabaseSizeMB > 0 {
+		batchArgs = append(batchArgs, []string{
+			databaseSizeFlag,
+			strconv.Itoa(a.storeConfig.DatabaseSizeMB),
+		}...)
+	}
+
+	// When mixed hdd/ssd devices are given, ceph-volume configures db lv on the ssd.
 	metadataDeviceSpecified := false
+	if a.metadataDevice != "" {
+		logger.Infof("using %s as metadataDevice and let ceph-volume lvm batch decide how to create volumes", a.metadataDevice)
```

Comment:
I know it's not the goal of the PR but it'd be nice to have this too: https://github.com/rook/rook/issues/2720
For clarify.

Reply:
@leseb actually that's in the todo

Reply:
Oops, I should have paid more attention!